### PR TITLE
avalanche forecast: show the correct logo in the back button

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Review by yuliadub — APPROVED** [review]

**Comment by stevekuznetsov** [comment]
Sometimes I see a lag with loading the image - however it's not clear if we can `Image.prefetch()` local assets?

**Comment by stevekuznetsov** [comment]
Found a workaround!

File: components/AvalancheCenterLogo.tsx [hunk 1]
```diff
@@ -96,3 +97,45 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
   actualStyle.aspectRatio = sizes[avalancheCenterId].width / sizes[avalancheCenterId].height;
   return images[avalancheCenterId](actualStyle);
 };
+
+export const preloadAvalancheCenterLogo = async (avalancheCenter: AvalancheCenterID) => {
```
Comment (floatplane) [inline components/AvalancheCenterLogo.tsx:101]: what does prefetching buy you here? these are local assets, not network ones

File: components/AvalancheCenterLogo.tsx [hunk 2]
```diff
@@ -96,3 +97,45 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
...
+    case 'MWAC':
+      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/MWAC.png')).uri);
+    case 'PAC':
+      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/PAC.png')).uri);
+    case 'SNFAC':
+      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/SNFAC.png')).uri);
+    case 'SAC':
+      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/SAC.png')).uri);
+    case 'WCMAC':
+      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/WCMAC.svg')).uri);
```
Comment (floatplane) [inline components/AvalancheCenterLogo.tsx:127]: do SVGs work in an `Image`? I kind of think they don't

**Review by floatplane — COMMENTED** [review]
couple of questions

File: components/AvalancheCenterLogo.tsx (hunk 1 above)
Comment (floatplane) [reply components/AvalancheCenterLogo.tsx:101]: (I guess you had already answered that question...surprised that it's slow to load, I wonder if that's a debug artifact)

Comment (stevekuznetsov) [reply components/AvalancheCenterLogo.tsx:101]: Yeah ... no idea how/why this was needed but it lagged/stuttered before.

File: components/AvalancheCenterLogo.tsx (hunk 2 above)
Comment (stevekuznetsov) [reply components/AvalancheCenterLogo.tsx:127]: Guess not - weird! We can fix this in the future, this PR does not introduce that logo.

**Comment by stevekuznetsov** [comment]
@floatplane hope the answers to your questions make sense!

**Pull request merged**